services/app/app/routes/portal.js

```diff
@@ -1,14 +1,24 @@
 import Route from '@ember/routing/route';
 import { queryManager } from 'ember-apollo-client';
+import { inject } from '@ember/service';
 import query from '@base-cms/company-update-app/gql/queries/portal';
 
 export default Route.extend({
+  config: inject(),
   apollo: queryManager(),
 
   async model({ hash }) {
     const variables = { input: { hash } };
     const model = await this.apollo.query({ query, variables, fetchPolicy: 'network-only' }, 'contentHash');
     if (!model) throw new Error('Invalid URL');
+    
+    // Check that leadershipEnable is true && if a leadershipCompay Label is set.  
+    // If so check if the label is in the list of company labels.
+    // If not set the LeadershipEnable to false.
+    if (this.config.leadershipEnabled && this.config.leadershipCompanyLabel) {
+      if(!model.labels.includes(this.config.leadershipCompanyLabel)) this.config.leadershipEnabled = false;
+    }
```

Comment:
Make this a computed property instead using the same fields. You can watch against the flag and label from the config and the model's labels to net the same result without the side effect of changing the config. You can then use that computed property in the template directly.

Something like 
```js
  leadershipEnabled: computed('config.{leadershipEnabled,leadershipCompanyLabel}', 'model.labels.[]', function() {
    // ex.; return config && label && model.labels.includes(label)
  }),
```

Just make sure to account for it being enabled without a required label as well :)